go.mod

```diff
@@ -3,9 +3,22 @@ module github.com/mattermost/mattermost-plugin-jenkins
 go 1.12
 
 require (
+	github.com/RackSec/srslog v0.0.0-20180709174129-a4725f04ec91 // indirect
```

Comment:
Would you please remove the indirect dependencies?

Reply:
Sure. I removed the indirect dependencies and commit it. Would you re-review it?